lib/ditcher.js

```diff
@@ -270,6 +270,22 @@ Ditcher.prototype.doList = function (params, callback) {
     options.sort = params.sort; //TOD: Should we validate more here? Hard to validate..
   }
 
+  // if _id is being queried, try convert the value to ObjectID hex string
+  if (query.hasOwnProperty("_id") && query._id.hasOwnProperty("$ne")) {
+    try {
+      query = { "_id": { "$ne": self.database.createObjectIdFromHexString(query._id.$ne) } };
+    } catch (err) {
+      // if it's not hex string then leave query as is
+      query = query;
```

Comment:
Is the reassignment even required? If there was an exception then `query` should still have it's original value.